flowmachine/tests/test_query_object_construction.py

```diff
@@ -355,3 +403,98 @@ def test_invalid_sampling_params_raises_error(sampling, message):
     with pytest.raises(ValidationError, match=message) as exc:
         _ = FlowmachineQuerySchema().load(query_spec)
     print(exc)
+
+
+@pytest.fixture
+def location_list_unmatched_agg():
```

Comment:
Looks like this new fixture isn't actually used?

Reply:
Oh yeah - turned out not to be useful.